dev/io.openliberty.concurrent.internal/src/io/openliberty/concurrent/internal/processor/ManagedExecutorDefinitionBinding.java

```diff
@@ -80,10 +101,12 @@ protected JNDIEnvironmentRefType getJNDIEnvironmentRefType() {
     public void merge(ManagedExecutorDefinition annotation, Class<?> instanceClass, Member member) throws InjectionException {
         final boolean trace = TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled();
         if (trace)
-            Tr.entry(this, tc, "merge", toString(annotation), instanceClass, member,
+            Tr.entry(this, tc, "merge", toString(annotation, eeVersion), instanceClass, member,
                      (XMLContextServiceRef ? "(xml)" : "     ") + "contextServiceRef: " + contextServiceJndiName + " << " + annotation.context(),
                      (XMLHungTaskThreshold ? "(xml)" : "     ") + "hungTaskThreshold: " + hungTaskThreshold + " << " + annotation.hungTaskThreshold(),
-                     (XMLMaxAsync ? "         (xml)" : "              ") + "maxAsync: " + maxAsync + " << " + annotation.maxAsync());
+                     (XMLMaxAsync ? "         (xml)" : "              ") + "maxAsync: " + maxAsync + " << " + annotation.maxAsync(),
+                     (XMLvirtual ? "          (xml)" : "               ") + "virtual: " + virtual + " << " + (eeVersion >= 11 ? annotation.virtual() : "Unspecified"),
+                     (XMLqualifers ? "        (xml)" : "            ") + "qualifiers: " + toString(qualifiers) + " << " + (eeVersion >= 11 ? toString(annotation.qualifiers()) : "Unspecified"));
```

Comment:
Even though we don't support these values prior to EE 11, would it be valuable to see if anything unexpected is coming in for them rather than showing Unspecified?  I'm wondering if Unspecified could mislead someone who is debugging issues

Reply:
If a user compiled their application using EE 11 and used the qualifiers attribute on their service annotation, but then ran with EE 10 I would assume we'd get a runtime exception when we attempted to load the class containing the annotation.  So I don't think we'd ever have any values coming in via the annotation, but rather we might get values coming in via the deployment descriptor.

Reply:
I used the word `Unspecified` since that matches what the injection binding superclass uses when no value was provided.  But in this case we could use something like `Unsupported` to make it clearer why no value was merged.